[2024-06-20T14:29:46.069+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T14:29:46.370+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.raw_orders.run manual__2024-06-20T14:29:01.398557+00:00 [queued]>
[2024-06-20T14:29:46.486+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.raw_orders.run manual__2024-06-20T14:29:01.398557+00:00 [queued]>
[2024-06-20T14:29:46.494+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-20T14:29:46.720+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtRunLocalOperator): transform.raw_orders.run> on 2024-06-20 14:29:01.398557+00:00
[2024-06-20T14:29:46.802+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4069) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T14:29:46.806+0000] {standard_task_runner.py:63} INFO - Started process 4102 to run task
[2024-06-20T14:29:46.808+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ELT-PROJECT', 'transform.raw_orders.run', 'manual__2024-06-20T14:29:01.398557+00:00', '--job-id', '604', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp0ut221in']
[2024-06-20T14:29:46.885+0000] {standard_task_runner.py:91} INFO - Job 604: Subtask transform.raw_orders.run
[2024-06-20T14:29:47.177+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-20T14:29:47.473+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT-PROJECT.transform.raw_orders.run manual__2024-06-20T14:29:01.398557+00:00 [running]> on host airflow-scheduler
[2024-06-20T14:29:48.265+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Data-Ninja' AIRFLOW_CTX_DAG_ID='ELT-PROJECT' AIRFLOW_CTX_TASK_ID='transform.raw_orders.run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-20T14:29:01.398557+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-20T14:29:01.398557+00:00'
[2024-06-20T14:29:48.283+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T14:29:48.312+0000] {local.py:174} INFO - Could not import dbtRunner. Falling back to subprocess for invoking dbt.
[2024-06-20T14:29:48.333+0000] {local.py:292} INFO - Cloning project to writable temp directory /tmp/tmpg1cc86aw from /opt/airflow/dags/dbt/dbt_project
[2024-06-20T14:29:48.404+0000] {local.py:303} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/ELT-PROJECT__transform/target/partial_parse.msgpack
[2024-06-20T14:29:48.634+0000] {base.py:84} INFO - Using connection ID 'snow_conn' for task execution.
[2024-06-20T14:29:48.640+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-06-20T14:29:48.656+0000] {base.py:225} INFO - Using real values for profile dbt_project
[2024-06-20T14:29:48.693+0000] {config.py:291} INFO - Creating temporary profiles.yml with use_mock_values=False at /tmp/tmpszj396i0/profiles.yml with the following contents:
dbt_project:
    outputs:
        dev:
            account: sk84886.europe-west2.gcp
            database: ELT_PROJECT
            password: '{{ env_var(''COSMOS_CONN_SNOWFLAKE_PASSWORD'') }}'
            role: SYSADMIN
            schema: PUBLIC
            type: snowflake
            user: DATANINJA
            warehouse: TRANSFORM
    target: dev

[2024-06-20T14:29:48.715+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpg1cc86aw', '--profiles-dir', '/tmp/tmpszj396i0', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpg1cc86aw
[2024-06-20T14:29:48.724+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-06-20T14:29:48.767+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpg1cc86aw', '--profiles-dir', '/tmp/tmpszj396i0', '--profile', 'dbt_project', '--target', 'dev']
[2024-06-20T14:29:48.892+0000] {subprocess.py:84} INFO - Command output:
[2024-06-20T14:30:06.906+0000] {subprocess.py:94} INFO - [0m14:30:06  Running with dbt=1.8.2
[2024-06-20T14:30:07.924+0000] {subprocess.py:94} INFO - [0m14:30:07  Updating lock file in file path: /tmp/tmpg1cc86aw/package-lock.yml
[2024-06-20T14:30:07.961+0000] {subprocess.py:94} INFO - [0m14:30:07  Installing dbt-labs/dbt_utils
[2024-06-20T14:30:09.531+0000] {subprocess.py:94} INFO - [0m14:30:09  Installed from version 0.8.6
[2024-06-20T14:30:09.540+0000] {subprocess.py:94} INFO - [0m14:30:09  Updated version available: 1.2.0
[2024-06-20T14:30:09.547+0000] {subprocess.py:94} INFO - [0m14:30:09
[2024-06-20T14:30:09.550+0000] {subprocess.py:94} INFO - [0m14:30:09  Updates available for packages: ['dbt-labs/dbt_utils']
[2024-06-20T14:30:09.551+0000] {subprocess.py:94} INFO - Update your versions in packages.yml, then run dbt deps
[2024-06-20T14:30:11.608+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-06-20T14:30:11.611+0000] {local.py:254} INFO - Update your versions in packages.yml, then run dbt deps
[2024-06-20T14:30:11.614+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'run', '--models', 'raw_orders', '--project-dir', '/tmp/tmpg1cc86aw', '--profiles-dir', '/tmp/tmpszj396i0', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpg1cc86aw
[2024-06-20T14:30:11.617+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-06-20T14:30:11.620+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'run', '--models', 'raw_orders', '--project-dir', '/tmp/tmpg1cc86aw', '--profiles-dir', '/tmp/tmpszj396i0', '--profile', 'dbt_project', '--target', 'dev']
[2024-06-20T14:30:11.697+0000] {subprocess.py:84} INFO - Command output:
[2024-06-20T14:30:24.679+0000] {subprocess.py:94} INFO - [0m14:30:24  Running with dbt=1.8.2
[2024-06-20T14:30:28.933+0000] {subprocess.py:94} INFO - [0m14:30:28  Registered adapter: snowflake=1.8.3
[2024-06-20T14:30:29.731+0000] {subprocess.py:94} INFO - [0m14:30:29  Unable to do partial parsing because profile has changed
[2024-06-20T14:30:50.037+0000] {subprocess.py:94} INFO - [0m14:30:50  [[33mWARNING[0m]: Did not find matching node for patch with name 'raw_employee_territores' in the 'models' section of file 'models/staging/schema.yml'
[2024-06-20T14:30:51.066+0000] {subprocess.py:94} INFO - [0m14:30:51  [[33mWARNING[0m]: Test 'test.dbt_project.not_null_raw_employee_territores_EMPLOYEEID.153e916ea2' (models/staging/schema.yml) depends on a node named 'raw_employee_territores' in package '' which was not found
[2024-06-20T14:30:51.072+0000] {subprocess.py:94} INFO - [0m14:30:51  [[33mWARNING[0m]: Test 'test.dbt_project.not_null_raw_employee_territores_TERRITORYID.3a9175e83e' (models/staging/schema.yml) depends on a node named 'raw_employee_territores' in package '' which was not found
[2024-06-20T14:30:53.277+0000] {subprocess.py:94} INFO - [0m14:30:53  Found 20 models, 27 data tests, 11 sources, 682 macros
[2024-06-20T14:30:53.299+0000] {subprocess.py:94} INFO - [0m14:30:53
[2024-06-20T14:31:02.366+0000] {subprocess.py:94} INFO - [0m14:31:02  Concurrency: 1 threads (target='dev')
[2024-06-20T14:31:02.368+0000] {subprocess.py:94} INFO - [0m14:31:02
[2024-06-20T14:31:02.403+0000] {subprocess.py:94} INFO - [0m14:31:02  1 of 1 START sql table model raw.raw_orders .................................... [RUN]
[2024-06-20T14:31:05.194+0000] {subprocess.py:94} INFO - [0m14:31:05  1 of 1 OK created sql table model raw.raw_orders ............................... [[32mSUCCESS 1[0m in 2.78s]
[2024-06-20T14:31:05.202+0000] {subprocess.py:94} INFO - [0m14:31:05
[2024-06-20T14:31:05.204+0000] {subprocess.py:94} INFO - [0m14:31:05  Finished running 1 table model in 0 hours 0 minutes and 11.88 seconds (11.88s).
[2024-06-20T14:31:05.448+0000] {subprocess.py:94} INFO - [0m14:31:05
[2024-06-20T14:31:05.451+0000] {subprocess.py:94} INFO - [0m14:31:05  [32mCompleted successfully[0m
[2024-06-20T14:31:05.452+0000] {subprocess.py:94} INFO - [0m14:31:05
[2024-06-20T14:31:05.454+0000] {subprocess.py:94} INFO - [0m14:31:05  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-06-20T14:31:09.345+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-06-20T14:31:09.346+0000] {local.py:254} INFO - [0m14:31:05  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-06-20T14:31:09.433+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-06-20T14:31:09.434+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-06-20T14:31:09.478+0000] {local.py:349} INFO - Inlets: [Dataset(uri='snowflake://sk84886.europe-west2.gcp/ELT_PROJECT.PUBLIC.orders', extra=None)]
[2024-06-20T14:31:09.481+0000] {local.py:350} INFO - Outlets: [Dataset(uri='snowflake://sk84886.europe-west2.gcp/ELT_PROJECT.raw.raw_orders', extra=None)]
[2024-06-20T14:31:09.484+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-20T14:31:12.924+0000] {dag.py:3954} INFO - Setting next_dagrun for ELT-PROJECT to 2024-06-20 00:00:00+00:00, run_after=2024-06-21 00:00:00+00:00
[2024-06-20T14:31:13.002+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/datasets/manager.py:53 SAWarning: DELETE statement on table 'task_outlet_dataset_reference' expected to delete 1 row(s); 0 were matched.  Please set confirm_deleted_rows=False within the mapper configuration to prevent this warning.
[2024-06-20T14:31:13.162+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T14:31:13.324+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT-PROJECT, task_id=transform.raw_orders.run, run_id=manual__2024-06-20T14:29:01.398557+00:00, execution_date=20240620T142901, start_date=20240620T142946, end_date=20240620T143113
[2024-06-20T14:31:13.470+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-20T14:31:13.537+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
